pages/portfolio.html

```diff
@@ -5,8 +5,61 @@
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Document</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/portfolio.css">
 </head>
+
 <body>
+  <div class="container-a">
+    <section id="section-1">
+      <h3>Find me on <a href="https://leetcode.com/lywyamy/">LeetCode</a> and <a href="https://github.com/lywyamy">GitHub</a></h3>
```

Comment:
In semantic HTML, heading tags should be used to indicate different sections of content, not as stylistic elements.  [Link](https://bighack.org/how-to-heading-structure-accessibility/)